docs/pages/machine-id/guides/ansible.mdx

```diff
@@ -6,30 +6,29 @@ description: Using Teleport with Ansible
 Ansible uses the OpenSSH client by default. Teleport supports SSH protocol and
 works as SSH jumphost.
 
-In this guide we will configure OpenSSH client to work with Teleport Proxy
-and run a sample ansible playbook.
+In this guide we will configure OpenSSH client to work with Machine ID
+and run a sample Ansible playbook.
 
 ## Prerequisites
 
-- Installed [Teleport Enterprise](../../enterprise/introduction.mdx) or [Teleport Cloud](../../cloud/introduction.mdx) >= (=teleport.version=)
-- `tsh` client tool  >= (=teleport.version=)
-- `ssh` openssh tool
-- `ansible` >= (=ansible.min_version=)
-- Optional tool `jq` to process `JSON` output.
+* Installed [Teleport Enterprise](../../enterprise/introduction.mdx) or [Teleport Cloud](../../cloud/introduction.mdx) >= (=teleport.version=)
+* `tbot` client tool  >= 9.0.0
+* `ssh` OpenSSH tool
+* `ansible` >= (=ansible.min_version=)
+* Optional tool `jq` to process `JSON` output
 
-## Step 1/3. Login and configure SSH
+## Step 1/3. Start Machine ID
 
-Log into Teleport with `tsh`:
+If you already have not done so, follow the
+[Machine ID Getting Started Guide](../../docs/pages/machine-id/getting-started.mdx)
+to create a bot user and start Machine ID.
 
-```code
-$ tsh login --proxy=example.com
-```
-
-Generate `openssh` configuration using `tsh config` shortcut:
+If you followed the above guide, you are interested in the
+`--destination-dir=/opt/machine-id` flag, which defines the directory where
+SSH certificates and OpenSSH configuration used by Ansible will be written to.
 
-```code
-$ tsh config > ssh.cfg
-```
+In particular, you will be using `/opt/machine-id/ssh_config` in Ansible
+configuration to define how Ansible should connect to Teleport nodes.
 
 <Admonition type="tip">
 You can edit matching patterns used in `ssh.cfg` if something
```

Comment:
I know this Admonition is from the original guide, but I'm not sure what this means. Is this saying that any error is probably due to a misconfigured host address match in the SSH client config? It might be okay to remove this since we have a Troubleshooting section at the end.